odoo/addons/base/ir/ir_qweb/qweb.py

```diff
@@ -1303,21 +1364,58 @@ def _compile_directive_call(self, el, options):
                 )
             )
 
+        # copy the original dict of options to pass to the callee
+        name_options = self._make_name('options')
```

Comment:
One should avoid cloning options every time, this is costly for the rendering time.